ias_proxy/sawtooth_ias_proxy/ias_proxy.py

```diff
@@ -194,7 +194,7 @@ def get_server():
         proxy_config = toml.loads(fd.read())
 
     # Verify the integrity (as best we can) of the TOML configuration file
-    valid_keys = set(['proxy_name', 'proxy_port', 'ias_url', 'spid_cert_file'])
+    valid_keys = set(['proxy_name', 'proxy_port', 'ias_url', 'ias_subscription_key'])
```

Comment:
Linter is complaining that this line is too long.

Reply:
have modified the file on the same commit

Reply:
@raneja14 there's a little more lint. You can check this locally by running: ```docker-compose -f ci/run-lint.yaml up```
The output right now looks like this: 
```lint_1    | ias_proxy/sawtooth_ias_proxy/ias_proxy.py:198:10: E128 continuation line under-indented for visual indent
lint_1    | -- pylint
lint_1    | ias_client
lint_1    | -- pycodestyle
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:50:80: E501 line too long (88 > 79 characters)
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:82:49: W291 trailing whitespace
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:83:80: E501 line too long (86 > 79 characters)
lint_1    | -- pylint
lint_1    | ************* Module sawtooth_ias_client.ias_client
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:50:0: C0301: Line too long (88/80) (line-too-long)
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:82:48: C0303: Trailing whitespace (trailing-whitespace)
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:83:0: C0301: Line too long (86/80) (line-too-long)
lint_1    | ias_client/sawtooth_ias_client/ias_client.py:21:0: W0611: Unused urljoin imported from urllib.parse (unused-import)
```
You can also see this in the jenkins logs, but relying on jenkins to check lint for you is not fun (as I rediscovered myself recently).